plugins/core/null/plugin-info

```diff
@@ -3,7 +3,12 @@
     "description": "Plugin that offers no configuration service and rejects TFTP/HTTP requests.",
     "capabilities": {
         "*, *, *": {
-            "sip.lines": 0
+            "lines": 0,
+            "ha": true,
+            "funckeys": 0,
+            "expansions": 0,
+            "switchboard": false,
+            "proto": "sip"
```

Comment:
Proposal
`proto` -> `protocol`
`ha` -> `high_availability`

Not a big deal
`expansions` -> `expansion_modules`
`funckeys` -> `function_keys`

Reply:
expansions = max count of expansion modules? ou max count of additional function keys?

Reply:
Is there a document explaining the different keys available?

Reply:
> `expansions` = max count of expansion modules? or max count of additional function keys?

Max count of expansion modules

Reply:
> Is there a document explaining the different keys available?

No, but I think that if the key names were self-explanatory it would not be necessary. In this case maybe `expansion_modules` should be named `max_expansion_modules` instead.

Reply:
If you name it `max_expansion_modules` then `funckeys` should be `max_function_keys`/`max_funckeys` and `lines` should be `max_lines`, etc.. ?

Reply:
I agree fblackburn1, there is no need for `max` in the key name, since we are in the `capabilities` section, values here give a maximum count.

My question went more in the direction of: if a key is missing, is there a reference that list all available keys? What if we don't know yet if the phone support `switchboard` for example? Should we use `null`? Or omit the key altogether? What if we add a new key in the future? Will we have to update all plugins at once to add the new key?

Reply:
There is no list to reference the available keys so it will be necessary to write documentation. When the support of a particular key is not sure, I think it would be ok to omit it. When adding a key, it would be nice if all the plugins were updated, but I think it may be too time consuming.